src/app/Instance/BridgeOverview/Components/OBDashboardTableView.css

```diff
@@ -0,0 +1,8 @@
+.OB-Dashboard__resource__truncated-string {
+  --pf-c-truncate--MinWidth: 0;
+  --pf-c-truncate__start--MinWidth: 0;
+}
+
+.OB-Dashboard__resource__labels {
+  padding: var(--pf-global--spacer--xs) 0;
+}
```

Comment:
Only applying vertical padding to the labels and switching to PF spacer variable.